### ISSUE TITLE
Rebranding PyLadies

### ISSUE THREAD

**Issue by Marie-L** [pyladies/project-communications#49]
Let's explore ideas regarding the redesign of the website and overall PyLadies brand
 
**Areas for discovery**

- [x] colour palette 

- [ ] PyLady

- [ ] PyLadies typeface

- [ ] pyladies.com UI designs

**Acceptance Criteria : Deliverables**

- [ ] colour palette sheet

- [ ] PyLady rebranding ideas

- [ ] Typeface rebranding ideas

- [ ] UI design

**Do you want to work on this issue (Yes/No)?**
 yes, but if anyone is interested in helping, please feel free to get involved 😊

cc / @BethanyG @AbigailDogbe 
cc / @pyladies/communications

**Comment by lorenanicole** [comment]
Could you clarify what these different palettes mean? Such as "classy" vs "matching"? Are we looking for input on which of these we like?

**Comment by Marie-L** [comment]
@lorenanicole the palettes were created by @BethanyG using [Color Space](https://mycolor.space/) following a conversation about branding and yes we are looking for input. This is the very beginning of our exploration but please share your feedback

**Comment by lorenanicole** [comment]
I very bias to blues, heh, so I'd vote for matching palette <3

**Comment by BethanyG** [comment]
Apologies for being so scattered.  I have more pallets, and will work on this during the June hack.

**Comment by BethanyG** [comment]
**Some Gradients:**


![PSF Blue and Yellow with PyLaides Red Gradients](https://user-images.githubusercontent.com/5923094/85230495-73ae4d00-b3a5-11ea-95f6-9a1935e3fb8a.png)

**Comment by BethanyG** [comment]
**PSF Yellow Base Color Schemes:**

![PSF Yellow Related](https://user-images.githubusercontent.com/5923094/85230508-888ae080-b3a5-11ea-9ac0-54e0c10f0ed0.png)

**Comment by BethanyG** [comment]
**More PyLadies Red Based Color Schemes:**

![PyLadies Red Related](https://user-images.githubusercontent.com/5923094/85230517-9ccedd80-b3a5-11ea-9be3-2233afaef9a2.png)

**Comment by BethanyG** [comment]
**PSF Blue Related Color Schemes:**

![PSF Blue Related](https://user-images.githubusercontent.com/5923094/85230528-afe1ad80-b3a5-11ea-8879-a0f15412e850.png)

**Comment by BethanyG** [comment]
**Finally, some Msc. Ones:**

![Other Msc  Color Schemes](https://user-images.githubusercontent.com/5923094/85230553-da336b00-b3a5-11ea-84a3-323c22139599.png)

**Comment by lorenanicole** [comment]
What could be cool is to get a vote on the coloring? Or, I guess, @Marie-L how should selection of color pallets be addressed? I feel as if if this group selected three pallets with a mock up those could be presented for a vote to the boarder global interim group?

**Comment by Marie-L** [comment]
@lorenanicole yes I agree, let's set up a poll. Yeah, a mock up alongside a palette would work. @BethanyG I'd like to learn how you design the Pylady, do you need any help ?

**Comment by BethanyG** [comment]
@Marie-L  - _**I'd love help!**_  Happy to show you how I set it up, and to work together on some mockups.   We could schedule some hack sessions - with us, project comms, and/or the larger community as well.  I have a `to-do` to write up some instructions for global/the chapter guide...but that will take me a while, so for this we might want to do online interactive sessions / Slack appointments.  We can also just start by working on/hading files to one another through our G-Drive. 

My  primary tool right now is [Sketch](https://www.sketch.com/).  It is mac-only, but [Lunacy](https://icons8.com/lunacy) is compatible and will open sketch files.  

I've also ported my main design files to [Affinity Designer](https://affinity.serif.com/en-us/designer/), which has versions for Mac, Windows and iPad.

For Linux (_works on Mac and Windows too!_), there is [Inkscape](https://inkscape.org/) - it has a bit of a learning curve, but definitely does the trick.  Sometimes borks other file formats, so you have to be a bit careful there.

When all else fails, individual assets can be saved out as SVG - and that will allow someone to edit them in any program that reads/writes SVG.

**Comment by Marie-L** [comment]
@BethanyG you're a star 🤩 ! Sounds great, I'll ping you to try and arrange a few sessions

**Comment by BethanyG** [comment]
As I start to go through these files again, I realize they're still not as organized as they could be.  Additionally, I've been looking at source control/version control options.  This is still prelim, but here are some sketch version control options:  https://45royale.com/blog/sketch-version-control/.   I think the one that is most in line with our current workflow which is GitHub based is [Kactus](https://kactus.io/) - but it is mac-only and sketch-only.  Just wanted to comment here.  As we dig deeper into this, we can open individual issues for specific things like Version Control, Colors, access, instructions, starter-packs, etc.

**Comment by Marie-L** [comment]
@BethanyG this is such a great idea! I really like Kactus, but let's give a few a try and also let's check whether our team members use mainly macs or PCs. Because if everyone is on a mac, we might as well go for Kactus

**Comment by BethanyG** [comment]
@Marie-L   -- curious as to why you closed this?  Are we resetting this entire repo and "tossing" prior work?

**Comment by Marie-L** [comment]
@BethanyG The rebranding process is mostly complete so I will open a new issue for 'Branding guidelines' , then will close this
